dashboard/app/controllers/admin_search_controller.rb

```diff
@@ -74,11 +74,29 @@ def undelete_section
   end
 
   def pilots
+    @pilots = Pilot.all
```

Comment:
This seems like kind of a strange place for these controller methods, why not create a pilots_controller file?

Reply:
Oh, because there were already pilot controller methods here. I guess not your problem :)